fe2-android/app/src/main/java/com/github/dedis/popstellar/repository/remote/GlobalNetworkManager.java

```diff
@@ -20,7 +20,7 @@
 @Singleton
 public class GlobalNetworkManager implements Disposable {
 
-  private static final String DEFAULT_URL = "ws://10.0.2.2:9000/organizer/client";
+  private static final String DEFAULT_URL = "ws://10.0.2.2:9000/client";
```

Comment:
Good idea to change this because it was confusing.

Reply:
Yeah, maybe in the past it was in sync but now it's just a waste of time as to test you have everytime to modify it